### ISSUE TITLE
Add support for Gitlab state backend

### ISSUE THREAD

**Issue by lamienois** [cloudskiff/driftctl#270]
**Description**

I would be great to support Gitlab and Azure blob as IaC sources in order to be able to scan remote state from them.

**Example**

```
$ driftctl scan --from tfstate+gitlab://MyProjectID/terraform/state/terraform.tfstate
```

**Comment by sjourdan** [comment]
Some background on GitLab's HTTP Terraform backend:  https://docs.gitlab.com/ee/user/infrastructure/terraform_state.html

**Comment by eliecharra** [comment]
Thanks for your interest for driftctl ! As I previously answered there https://github.com/cloudskiff/driftctl/issues/176#issuecomment-770721290 you can use `terraform state pull` as a workaround at the moment 😉

**Comment by sjourdan** [comment]
Hey @lamienois can you confirm that the newly released driftctl v0.7.0 with HTTP backend support fixes your use case? (git ref is https://github.com/cloudskiff/driftctl/issues/176) 

If it does, we should add a gitlab example soon (https://docs.driftctl.com/0.7.0/usage/cmd/scan-usage#--from)

**Comment by sundowndev** [comment]
The example is now live on the main branch https://docs.driftctl.com/next/usage/cmd/scan-usage#http--gitlab

**Comment by sundowndev** [comment]
Closing this since it has been fixed in v0.7.0. @lamienois feel free to reopen if you encounter issues with the HTTP backend.
